### PR TITLE
added Nov19 seminar

### PR DESCRIPTION
**Pull request by mozhgan-kch**

### CODE REVIEW

File: _events/seminar-2019-11-26.md [hunk 1]
```diff
@@ -8,10 +8,12 @@ to: "13:00"
...
+title: "Object Oriented Programming in MATLAB?"
 
 image:
 slides_url:
 ---
 
-**Abstract**: "TBA"
+**Abstract**: In the context of MATLAB, Object-oriented programming (OOP) offers a convenient way of creating new capabilities that are easily customised to specific needs. My talk will describe the basics of how OOP is handled in MATLAB, some selected cases where OOP is used within MATLAB itself, and then some examples, mostly from my personal experience in image computing, to illustrate how turning to OOP can help achieve two core outcomes - streamline the implementation of new features and simplify the way they are accessed by other users.
+
+**Bio**: Alberto Biancardi is an  Image Computing Scientist in the POLARIS group, Department of Infection, Immunity & Cardiovascular Disease, The University of Sheffield. His focus is on the analysis of images of the lungs, both structural and functional, reflecting the group's expertise in the use of hyperpolarised noble gases in magnetic resonance imaging.
```
Comment (Robadob) [inline _events/seminar-2019-11-26.md]: Double space after 'an'

File: _events/seminar-2019-11-26.md [hunk 2]
```diff
@@ -8,10 +8,12 @@ to: "13:00"
...
 institute: "University of Sheffield"
-title: "TBA"
+title: "Object Oriented Programming in MATLAB?"
 
 image:
 slides_url:
 ---
 
-**Abstract**: "TBA"
+**Abstract**: In the context of MATLAB, Object-oriented programming (OOP) offers a convenient way of creating new capabilities that are easily customised to specific needs. My talk will describe the basics of how OOP is handled in MATLAB, some selected cases where OOP is used within MATLAB itself, and then some examples, mostly from my personal experience in image computing, to illustrate how turning to OOP can help achieve two core outcomes - streamline the implementation of new features and simplify the way they are accessed by other users.
```
Comment (Robadob) [inline _events/seminar-2019-11-26.md]: `My talk`, perhaps better worded as 'This talk`

**Review by Robadob — COMMENTED** [review]
Few minor things.

File: _events/seminar-2019-11-26.md (hunk 2 above)
Comment (mozhgan-kch) [reply _events/seminar-2019-11-26.md]: Done

**Review by Robadob — APPROVED** [review]

**Pull request merged**